website/docs/docs/bi-directional-contract-testing/contracts/oas/keyword-support.md

```diff
@@ -177,13 +177,13 @@ These are able to work because the defined schemas are "open" by default. What d
 
 From https://json-schema.org/understanding-json-schema/reference/object.html#additional-properties:
 
-> The [`additionalProperties`](https://json-schema.org/understanding-json-schema/reference/object.html#additional-properties) keyword is used to control the handling of extra stuff, that is, properties whose names are not listed in the `properties` keyword or match any of the regular expressions in the `patternProperties` keyword. By default any additional properties are allowed.
+> The [`additionalProperties`](https://json-schema.org/understanding-json-schema/reference/object.html#additional-properties) keyword controls the handling of extra stuff, that is, properties whose names are not listed in the `properties` keyword or match any of the regular expressions in the `patternProperties` keyword. Additional properties are allowed by default.
```

Comment:
Just a note, that this is a direct quote from the website referenced above.

Reply:
I have edited and restored the old version of the statement.